main.js

```diff
@@ -0,0 +1,200 @@
+      tdxConfig,
+      secret: storedSecret,
+      token: storedToken,
+      timeout: appConfig.scraperTimeout,
+    });
+
+    let output;
+    switch (commandName) {
+      case "signin":
+        await commandHandler.handleSignin(argumentSecret);
+
+        setEnv(TDX_CURRENT_ALIAS, aliasToEnv(alias));
+        // Store the argument secret
+        if (argumentSecret.id) setEnv(getSecretAliasName(alias), jsonToBase64(argumentSecret));
+        setEnv(getTokenAliasName(alias), commandHandler.getToken());
+        console.log("OK");
+        break;
+      case "signout":
+        commandHandler.handleSignout();
+        setEnv(getTokenAliasName(alias), "");
+        setEnv(getSecretAliasName(alias), "");
+        break;
+      case "info":
+        output = await commandHandler.handleInfo({id, type});
+        console.log(output);
+        break;
+      case "config":
+        console.log(tdxConfig);
+        break;
+      case "list":
+        output = {
+          default: alias,
+          aliases: listAliases(appConfig.tdxConfigs),
+        };
+        console.log(output);
+        break;
+      case "runapi":
+        output = await commandHandler.handleRunApi({
+          command,
+          apiArgs: commandProps.apiArgs,
+          apiArgsStringify: commandProps.apiArgsStringify,
+        });
+        console.log(JSON.stringify(output, null, 2));
+        break;
+      case "download":
+        await commandHandler.handleDownload(id, filepath);
+        break;
+      case "upload":
+        output = await commandHandler.handleUpload(id, filepath);
+        console.log(output);
+        break;
+      case "copyalias":
+        await copyAliasConfig({
+          appConfig,
+          alias,
+          aliasName,
+          configFileName: "./config.json",
+        });
+        console.log("OK");
+        break;
+      case "modifyalias":
+        const aliasConfig = await readJsonFromFile(configJson);
+        await modifyAliasConfig({
+          appConfig,
+          modifyAlias: aliasName,
+          aliasConfig,
+          configFileName: "./config.json",
+        });
+        console.log("OK");
+        break;
+      case "removealias":
+        if (alias === aliasName) throw Error("Can't remove the running alias.");
+        await removeAliasConfig({
+          appConfig,
+          aliasName,
+          configFileName: "./config.json",
+        });
+        console.log("OK");
+        break;
+      case "abortdatabot":
+        output = await commandHandler.handleAbortDatabot(instanceId);
+        console.log(output);
+        break;
+      case "stopdatabot":
+        output = await commandHandler.handleStopDatabot(instanceId);
+        console.log(output);
+        break;
+      case "startdatabot":
+        const functionPayload = await readJsonFromFile(configJson);
+        output = await commandHandler.handleStartDatabot(databotId, functionPayload);
+        console.log(output);
```

Comment:
Why not just have a single console.log(output) at the end of the case statement?

Reply:
Very good point.